central/vulnmgmt/vulnerabilityrequest/manager/requestmgr/manager_impl_flat_cve_data_test.go

```diff
@@ -54,14 +56,26 @@ func (m *managerImplTestFlatData) SetupSuite() {
 	var err error
 	m.vulnReqDatastore, err = vulnReqDatastore.GetTestPostgresDataStore(m.T(), m.testDB, cache.New(), cache.New())
 	m.Require().NoError(err)
-	imageStore := imageDS.NewWithPostgres(
-		imagePostgresV2.New(m.testDB.DB, false, concurrency.NewKeyFence()),
-		mockRisk,
-		ranking.ImageRanker(),
-		ranking.ComponentRanker(),
-	)
+	var imageStore imageDS.DataStore
+	var imageV2Store imageV2DS.DataStore
+	if features.FlattenImageData.Enabled() {
+		imageV2Store = imageV2DS.NewWithPostgres(
```

Comment:
Do you also have to change where the imageStore is used to upsert or get test images?